plugin/src/main/java/com/denizenscript/denizen/objects/LocationTag.java

```diff
@@ -4079,6 +4080,39 @@ else if (material.hasModernData() && material.getModernData() instanceof org.buk
             return new ElementTag(BukkitScriptEvent.inCheckInternal(attribute.context, "is_in tag", object, attribute.getParam(), "is_in tag", "is_in tag"));
         });
 
+        // <--[tag]
+        // @attribute <LocationTag.last_interacted_slot>
+        // @returns ElementTag(Number)
+        // @description
+        // Returns the last interacted slot of a Chiseled Bookshelf inventory.
+        // -->
+        tagProcessor.registerTag(ElementTag.class, "last_interacted_slot", (attribute, object) -> {
```

Comment:
Same for these (`last_interacted_slot` seems to be 1.19+, `slot` 1.20+) - `LocationTag` has existing if checks for both of these versions